test/core/modules/resize.js

```diff
@@ -1,4 +1,15 @@
 const testModule = require('../templates/module-test'),
   benchmark = 'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABUAAAAVCAYAAACpF6WWAAAAAklEQVR4AewaftIAAAAzSURBVLXBAQEAMAiAME7/zN4Ssr2BzzEJSEACEpCABCQgAQlIQAISkIAEJCABCUhAAhJYclMCJyy7k2QAAAAASUVORK5CYII=';
 
-testModule('resize', {resize: '129%'}, benchmark);
\ No newline at end of file
+options1 = {
+  resize: '70.85%'
+},
+options2 = {
+  resize: '60 %'
+};
+options3 = {
+  resize: '90'
+};
+testModule('resize', {resize: '129%'}, benchmark);
+
+require('../templates/options-test')('resize', [option1, option2, option3], [benchmark, benchmark, benchmark]);
```

Comment:
If you resize by different values, shouldn't the benchmark images be different too?

Reply:
Hi, can you help me generate images for the benchmark? I tried some base64 png's but the options' test is failing. However, there is no issue with the parsing of percentages. So I assume it is with the way I serve images.

Reply:
You can generate the images via CLI (read the readme/docs), or in the [browser](https://beta.sequencer.publiclab.org) and then convert them to a data URI using an online tool (google it, its very easy :))